docs/index.txt

```diff
@@ -45,13 +45,25 @@ Each user should also :ref:`install the Determined command-line tools <install-c
 
 Next Steps
 """"""""""
+
 To get started using Determined, check out the :ref:`quick-start`.
 
-Next, follow the tutorial for your preferred deep learning framework:
+Determined has two Experiment APIs: Trial and Native.  With the :ref:`model-definitions_trial-api`, developers expose the components of a model definition to Determined’s training loop implementation.  With the :ref:`model-definitions_native-api` (TensorFlow Keras and Estimators), developers instrument training scripts adhering to these frameworks' native patterns including their training loops.
```

Comment:
blocking: I would list these in the same order you describe them in the next paragraphs, and I'd put Native in front of Trial.

Reply:
Also, not clear to me if this should be done _after_ quick-start, or is this if you don't want to use quick start? Would this go at the end of the quick-start guide too?

Reply:
Okay, I won't block, but I think making it a bit more narrative will help. "We recommend you follow our [Quick Start](url) guide if you're new to Determined. Next, learn more about our different APIs and how to interact with them." (or something like this)

Reply:
done